OpenSim/Tests/Wrapping/testWrapCylinder.cpp

```diff
@@ -0,0 +1,332 @@
+/* -------------------------------------------------------------------------- *
+ *                         OpenSim:  testWrapCylinder.cpp                     *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2022 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include <OpenSim/OpenSim.h>
+#include <OpenSim/Auxiliary/auxiliaryTestFunctions.h>
+
+#include <string>
+#include <iostream>
+#include <cmath>
+#include <sstream>
+#include <functional>
+#include <memory>
+
+using namespace OpenSim;
+
+namespace {
+
+    // A path segment determined in terms of the start and end point.
+    struct PathSegment final {
+        PathSegment() = default;
+
+        PathSegment(
+            const SimTK::Vec3& startPoint,
+            const SimTK::Vec3& endPoint) :
+            start(startPoint),
+            end(endPoint)
+        {}
+
+        SimTK::Vec3 start {SimTK::NaN};
```

Comment:
Initializers don't usually have spaces between the name and the initializing statement (e.g. `std::vector<int> vals(2);`, `std::vector<int> vals{1,2,3};`, not `std::vector<int> vals (2);`)